tests/src/core/testqgsogcutils.cpp

```diff
@@ -368,6 +369,14 @@ void TestQgsOgcUtils::testExpressionFromOgcFilter_data()
                                   "</Intersects>"
                                   "</Filter>" )
                                 << QStringLiteral( "intersects($geometry, geom_from_gml('<Point><coordinates>123,456</coordinates></Point>'))" );
+
+  std::unique_ptr<QgsVectorLayer> layer = qgis::make_unique<QgsVectorLayer>( "Point?crs=epsg:4326&field=name:string(20)&index=yes", "temporary", "memory" );
```

Comment:
Am I missing something or `layer` is not useful here?

Reply:
Good catch, fixed